apple_passkit/pubspec.yaml

```diff
@@ -25,6 +25,3 @@ flutter:
     platforms:
       ios:
         pluginClass: ApplePasskitPlugin
-
-platforms:
```

Comment:
Is this no longer needed? I thought this needs to be added to specify that this plugin only supports iOS?

Reply:
this is not needed since platforms are already defined above and that should be enough, see [here](https://docs.flutter.dev/packages-and-plugins/developing-packages#plugin-platforms)

Reply:
Oh that's nice